systemtest/pom.xml

```diff
@@ -222,6 +222,12 @@
             <artifactId>jackson-databind</artifactId>
             <version>${fasterxml.jackson-databind.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.strimzi</groupId>
+            <artifactId>test-containers</artifactId>
+            <version>0.18.0-SNAPSHOT</version>
```

Comment:
Is the version needed here in this place?

Reply:
this should be there...${project.version}